test/clj_sqs_extended/core_test.clj

```diff
@@ -0,0 +1,38 @@
+(ns clj-sqs-extended.core-test
+  "Basic tests for the primary API of `clj-extended-sqs`."
+  (:require [clojure.test :refer [deftest is testing use-fixtures]]
+            [clj-sqs-extended.core :as sqs-ext]
+            [clj-sqs-extended.test-helpers :as helpers]
+            [clj-sqs-extended.test-fixtures :as fixtures]))
+
+
+(use-fixtures :once fixtures/with-localstack-environment)
+
+(def ^:private test-message "What hath God wrought?")
+(def ^:private test-message-large (helpers/random-string-with-length 300000))
```

Comment:
large is subjective. perhaps spell it out and call this `test-message-larger-than-256kb` for clarity.